client/client.go

```diff
@@ -178,11 +178,11 @@ func (c *Client) Handle(ph ProposalHandler, uh UpdateHandler) {
 		case *VirtualChannelProposal:
 			go c.handleChannelProposal(ph, env.Sender, msg)
 		case *msgChannelUpdate:
-			go c.handleChannelUpdate(uh, env.Sender, msg)
+			go c.handleChannelUpdate(context.TODO(), uh, env.Sender, msg)
 		case *virtualChannelFundingProposal:
-			go c.handleChannelUpdate(uh, env.Sender, msg)
+			go c.handleChannelUpdate(context.TODO(), uh, env.Sender, msg)
 		case *virtualChannelSettlementProposal:
-			go c.handleChannelUpdate(uh, env.Sender, msg)
+			go c.handleChannelUpdate(context.TODO(), uh, env.Sender, msg)
```

Comment:
Oh right, this would only make a difference if the client gets closed while there is an on-going request, but i will revert it.

Reply:
I suggest we postpone context linting to a separate PR.

Reply:
Yes i also did not like the ctx changes, will remove the liner for now. Otherwise it wants to have `nolint` at every call in the tree where a ctx could be passed, that is annoying as well.  
If we instead remove the ctx from the `relay.Cache` it would be fixed.